installer/README.md

```diff
@@ -1,29 +1,71 @@
-# Introduction
-This is a standalone/non-containerized installer for SODA Infrastructure Manager (delfin) project.
-It contains a script and options to check the environment feasible for installing delfin. Installs required dependent software/binaries.
+# Delfin Installation Guide
+
+The SODA Delfin supports two types of installation
+* Installation using Ansible for user experiance with Dashboard
+* Installation using scripts
```

Comment:
using scripts? Or Installation with Dashboard and Installation with CLI?

Reply:
Removed dashboard, srm_toolchain and auth install files. Updated banner and documentation.